README.md

```diff
@@ -15,49 +15,105 @@
   </strong>
 </div>
 
+&nbsp;
 
-## Introduction
-[Enthusiast](https://upsidelab.io/tools/enthusiast) is an open-source toolkit for building AI-powered agentic workflows. It provides tools like Retrieval-Agumented Generation (RAG), vector search and workflow orchestrator. You can connect it to cloud-based models, like OpenAI, as well as self-hosted deployments of Mistral, LLaMA or Deepseek.
+[Enthusiast](https://upsidelab.io/tools/enthusiast) is an open-source, customizable agentic AI toolkit for building AI agents that automate e‑commerce workflows and human-intensive back‑office tasks. It combines retrieval‑augmented generation (RAG), vector search, and workflow orchestration to give teams a turnkey starting point to build custom AI agents grounded in their product data and processes — agents that can run in minutes and scale to production.
 
-Enthusiast helps you cut down time, cost and complexity when building AI-powered tools with customized agents - for smart information search, customer support, content generation or knowledge base automation. Enthusiast provides validation and evaluation components, to reduce hallucinations and ensure responses are grounded in actual data.
+Enthusiast gives you full control over infrastructure, data, models, prompts, and integrations. You can run it locally with Docker, deploy to Kubernetes, or host it on-premise. Its model-agnostic architecture supports both cloud-based models like OpenAI and Gemini, and self-hosted deployments such as Mistral, Ollama, or DeepSeek.
```

Comment:
`You can run it locally with Docker` Do you think mentioning local setup is needed there?